podcasts/End of Year/Stories/TopOnePodcastStory.swift

```diff
@@ -21,20 +25,57 @@ struct TopOnePodcastStory: ShareableStory {
         let podcast = topPodcast.podcast
         GeometryReader { geometry in
             PodcastCoverContainer(geometry: geometry) {
-                PodcastStackView(podcasts: [podcast], geometry: geometry)
-
                 StoryLabelContainer(geometry: geometry) {
                     let title = podcast.title ?? ""
-                    let author = podcast.author ?? ""
-                    StoryLabel(L10n.eoyStoryTopPodcast("\n" + title, author), highlighting: [title, author], for: .title)
+                    StoryLabel(L10n.eoyStoryTopPodcast(title), for: .title, geometry: geometry)
 
                     let time = topPodcast.totalPlayedTime.storyTimeDescription
-                    let count = L10n.eoyStoryListenedToEpisodesText(topPodcast.numberOfPlayedEpisodes)
-                    StoryLabel(L10n.eoyStoryTopPodcastSubtitle(topPodcast.numberOfPlayedEpisodes, time), highlighting: [time, count], for: .subtitle)
-                        .opacity(0.8)
+                    StoryLabel(L10n.eoyStoryTopPodcastSubtitle(topPodcast.numberOfPlayedEpisodes, time), for: .subtitle, color: Color(hex: "8F97A4"), geometry: geometry)
                 }
-            }
-        }.background(DynamicBackgroundView(podcast: podcast))
+
+                ZStack {
+                    podcastCover(1)
+                        .frame(width: geometry.size.width * 0.3, height: geometry.size.width * 0.3)
```

Comment:
Maybe you're already thinking "EXTRACT THOSE TO CONSTANTS!". Ok, ok, I get it!

I leave it this way because the numbers are so many and so different that leaving it here might be better than extracting it. Or not?